Automattic/wp-e2e-tests
# Remove visdiffs from test specs

The first step to removing the visual diff tests from our codebase (https://github.com/Automattic/wp-e2e-tests/issues/1424). This removes the visual diff steps from the individual test specs.

To test:
1. Run `./run.sh -v` to confirm that no tests are run (no tests tagged `@visdiff`).
2. Confirm that other tests run as expected (e.g. with `./run.sh -g`).
